src/TwigComponent/doc/index.rst

```diff
@@ -676,6 +676,47 @@ the subdirectory:
     {# renders as: #}
     <button class="primary">Click Me!</button>
 
+If your component lives in a directory with the same name, you can
```

Comment:
I would emphasis "anonymous" somewhere in that sentence. I know it's written a couple lines before, but I would not someone miss-read this feature is available for class-based or Live components :)